webpack.config.js

```diff
@@ -9,18 +9,27 @@ const RemoveEmptyScriptsPlugin = require( 'webpack-remove-empty-scripts' );
 /**
  * WordPress dependencies
  */
-const defaultConfig = require( '@wordpress/scripts/config/webpack.config' );
+const { getAsBooleanFromENV } = require( '@wordpress/scripts/utils' );
+
+const hasExperimentalModulesFlag = getAsBooleanFromENV( 'WP_EXPERIMENTAL_MODULES' );
+let scriptConfig, moduleConfig;
+
+if ( hasExperimentalModulesFlag ) {
+	[ scriptConfig, moduleConfig ] = require( '@wordpress/scripts/config/webpack.config' );
+} else {
+	scriptConfig = require( '@wordpress/scripts/config/webpack.config' );
+}
```

Comment:
```suggestion
const [ scriptConfig, moduleConfig ] = require( '@wordpress/scripts/config/webpack.config' );
```

Since you have already enabled the experimental modules in the build script, there is no need to re-determine the logic in the webpack configuration. The `getAsBooleanFromENV('WP_EXPERIMENTAL_MODULES')` function will always return true in this context.

Reply:
- [x] Updated the imports ✅